### PR TITLE
Handle foreman_jobs_service being a required parameter

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by mmoll** [comment]
test fails

File: config/foreman.migrations/180821005330_foreman_jobs_service.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if answers['foreman'].is_a?(Hash) && answers['foreman']['foreman_jobs_service'].nil?
+  answers['foreman']['foreman_jobs_service'] = 'dynflowd'
```
Comment (ehelms) [inline config/foreman.migrations/180821005330_foreman_jobs_service.rb]: From testing this should just be `jobs_service`. Please also open one for Katello :)

**Review by ehelms — APPROVED** [review]
Upgrade successful after applying this patch.

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**